modules/ROOT/pages/index.adoc

```diff
@@ -118,38 +116,37 @@ a|
 | API Platform | API Proxy Builder Service | v1.2.52
 | API Platform | RAML Parser Service | v0.3.76
 | API Platform | Proxies XAPI Service | v1.2.75
-| API Designer | API Designer XAPI | v3.17.0
-| API Designer | API Designer UI | v2.18.2
-| Mocking Service | Mocking Service | v2.15.2
-| Mozart | Design Center UI | v2.0.69
-| Mozart | Mozart XAPI | v2.0.216
-| Mozart | Mozart Project Manager Service | v3.0.20
-| Runtime Manager | Cloudhub MCM | 
+| API Designer | API Designer XAPI | v3.29.0
+| API Designer | API Designer UI | v2.32.0
+| Mocking Service | Mocking Service | v2.29.1
+| Mozart | Design Center UI | v2.0.109
+| Mozart | Mozart XAPI | v2.0.232
+| Mozart | Mozart Project Manager Service | v3.0.45
+| Runtime Manager | Cloudhub MCM | v2.17.34
 | Runtime Manager | Cloudhub UI | 
-| Runtime Manager | AMC UI Rest Facade | 
-| Runtime Manager | Hybrid Rest | 
-| Runtime Manager | Metadata Manager |
+| Runtime Manager | AMC UI Rest Facade | v2.12.25
+| Runtime Manager | Hybrid Rest | v2.31.159
+| Runtime Manager | Metadata Manager | 
 | Runtime Manager | Platform Alerts | 
-| Runtime Manager | Mule Message Processor | 
+| Runtime Manager | Mule Message Processor | v2.0.37
 | Runtime Manager | Platform Messages | 
-| Access Management | Authentication Server | v0.81.x
+| Access Management | Authentication Server | v1.2.48
 | Access Management | Core Services UI | v0.81.x
 | Access Management | Anypoint Signin | v0.81.x
 | Access Management | Audit Log | v0.81.x
 | Access Management | Landing Page UI | v0.81.x
 | Access Management | Navbar | v0.81.x
-| Anypoint Platform PCE | Anypoint Platform PCE packaging | v3.0.0
-| API Modeling Framework (AMF) | AMF Service | v1.0.152
-| VCS | VCS  | v0.3.1420
-| VCS | VCS Git Proxy | v0.2.85
-| Anypoint Studio | Studio | v7.7.0
+| Anypoint Platform PCE | Anypoint Platform PCE packaging | v3.1.0
```

Comment:
lowercase "packaging"?

I'm just skimming these terms for inconsistencies: they are a mix of actual names and...just things, but they seem to use mostly title-style capitalization.